### DISCUSSION TITLE
Release cadence?

### DISCUSSION THREAD

**Discussion by mountainaireman** [cloud-carbon-footprint/cloud-carbon-footprint#1201] [General]
Is there a planned release cadence?  If so, is that publicly documented anywhere?  I am eagerly waiting for the next release as I am hoping it will get my Azure regions to no longer be unknown.

**Comment by ccasher** [comment]
Hi @mountainaireman - just answered this question [here](https://groups.google.com/g/cloud-carbon-footprint/c/nHjDitPEBnI) in the google group thread!
